controllers/operator/nmstate_controller_test.go

```diff
@@ -371,6 +372,44 @@ var _ = Describe("NMState controller reconcile", func() {
 		})
 	})
 
+	Context("XXX CHOCOBOMB", func() {
```

Comment:
?

Reply:
It stays here for now because PR is on-hold because I need to clarify with API folks if we want to expose it or not. Basically some corporate yadayada because this feature depends on another feature but this other feature won't be developed before our deadline to develop our feature 🤷🏻

Reply:
Fixed